frontend/src/locales/scripts/en.json5

```diff
@@ -826,5 +826,43 @@
       source: "Open source site",
       creator: "Open creator page",
     },
+    resultCountLabel: {
+      creator: {
+        audio: {
+          zero: "No audio files by this creator in {source}",
```

Comment:
Is "track" a better word than "file" here? I would like us to identify a word that's more descriptive than "file" and use it consistently in the UI.